src/utils.ts

```diff
@@ -50,7 +50,7 @@ export function toWinDirFormat(dir: string = ''): string {
 
 export function getLocalAppDataPath(): string {
   const path = process.env.PATH;
-  const userRegExp = /\/mnt\/([a-z])\/Users\/(.+?)\/AppData\//;
+  const userRegExp = /\/mnt\/([a-z])\/Users\/([^\/]+)\/AppData\//;
```

Comment:
not sure if we need to be able to span a few directories deep 🤷‍♂️

Reply:
as far as I know we don't, we just need to find the user and Drive letter.
/mnt/**c**/Users/**wardpeet**/AppData/

Reply:
ah ok, sg then

let's go ahead and throw the colon in the regex too to make sure it doesn't span the separator either :)